pkg/cli/cli.go

```diff
@@ -717,13 +747,28 @@ func getCompleter(completer string) *readline.PrefixCompleter {
 		readline.PcItem("upload"),
 	)
 
+	//generate menu
+	var generate = readline.NewPrefixCompleter(
+		readline.PcItem("set",
+			readline.PcItemDynamic(serv.GetAgentOptionStrings()),
+			readline.PcItem("Goos", readline.PcItemDynamic(util.ValidGoos)),
```

Comment:
We're going to need to check and make sure Go is installed on the OS too. Having Go installed isn't a prerequisite to using Merlin.

Reply:
👍 , I had this in the plan, will introduce once everything else is a bit more stable